CMakeLists.txt

```diff
@@ -28,6 +28,22 @@ endif()
 # set the project name
 project(velox)
 
+# If we are in an active conda env disable search in system paths and add env to
+# prefix path
+if(DEFINED ENV{CONDA_PREFIX})
+  if(NOT DEFINED ENV{VELOX_DEPENDENCY_SOURCE} OR "$ENV{VELOX_DEPENDENCY_SOURCE}"
+                                                 STREQUAL "CONDA")
+    message(STATUS "Using Conda environment: $ENV{CONDA_PREFIX}")
+    set(CMAKE_FIND_USE_SYSTEM_ENVIRONMENT_PATH FALSE)
+    list(APPEND CMAKE_PREFIX_PATH "$ENV{CONDA_PREFIX}")
+    # Override in case it was set to CONDA
+    set(ENV{VELOX_DEPENDENCY_SOURCE} AUTO)
```

Comment:
nit: why do we need to set VELOX_DEPENDENCY_SOURCE to AUTO if it was set as CONDA ?

Reply:
Because resolve_dependency has no code path for  CONDA (the only change is the prefix path at this time), so it would error if we don't reset it. We could also set it to SYSTEM but that would than require special treatment for the deps we can't add via conda (folly at the moment, in the future fbthrift, fizz, wangle)